utils/rpms/daos.spec

```diff
@@ -284,7 +284,6 @@ getent group daos_admins >/dev/null || groupadd -r daos_admins
 %{_bindir}/cart_ctl
 %{_bindir}/self_test
 %{_bindir}/dmg
-%{_bindir}/dmg_old
```

Comment:
Changes such as this that will remove a binary from the RPM package should update the Release number (near the top of the file) and provide a corresponding %changelog entry lower in the file.

Reply:
@kccain looks like the way I update the relaese info is not correct, can you please comment what is not right?
Maybe the %changelog timestamp and email info should be auto generated?

Reply:
The date format in the changelog may be the problem (June in the changelog entry you added, vs Jun that I suspect is needed, 3 letter abbreviation of the month). I discovered this tool that may be helpful in starting from the unmodified daos.spec and bumping the Release number and adding a corresponding changelog entry

I ran it locally and then examined the diffs

$ cd daos/utils/rpms
$ rpmdev-bumpspec --comment="Remove dmg_old from DAOS RPM package" --userstring="Hua Kuang <hua.kuang@intel.com>" daos.spec 
$ git diff
diff --git a/utils/rpms/daos.spec b/utils/rpms/daos.spec
index 1799a67..b3f4718 100644
--- a/utils/rpms/daos.spec
+++ b/utils/rpms/daos.spec
@@ -6,7 +6,7 @@
 
 Name:          daos
 Version:       1.1.0
-Release:       19%{?relval}%{?dist}
+Release:       20%{?relval}%{?dist}
 Summary:       DAOS Storage Engine
 
 License:       Apache
@@ -356,6 +356,9 @@ getent group daos_admins >/dev/null || groupadd -r daos_admins
 %{_libdir}/*.a
 
 %changelog
+* Wed Jun 03 2020 Hua Kuang <hua.kuang@intel.com> - 1.1.0-20
+- Remove dmg_old from DAOS RPM package
+